cpp-unisocket/main.cpp

```diff
@@ -1,27 +1,20 @@
-#include <iostream>
 #include <hazelcast/client/hazelcast_client.h>
-#include <thread>
-#include <chrono>
 
 int main() {
     hazelcast::client::client_config config;
     config.get_network_config().use_public_address(true)
                             .add_address(hazelcast::client::address{"<EXTERNAL-IP>", 5701})
                             .set_smart_routing(false);
     auto hz = hazelcast::new_client(std::move(config)).get();
-
     std::cout << "Successful connection!" << std::endl;
     std::cout << "Starting to fill the map with random entries." << std::endl;
-
     auto map = hz.get_map("map").get();
     while (true) {
         int random_key = rand() % 100000;
         try {
             map->put("key-" + std::to_string(random_key), "value-" + std::to_string(random_key));
-            if (random_key % 100 == 0) {
-                std::cout << "Current map size: " + std::to_string(map->size().get()) << std::endl;
-                std::this_thread::sleep_for(std::chrono::seconds(1));
-            }
+            std::cout << "Current map size: " + std::to_string(map->size().get()) << std::endl;
+            std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
Most client examples, except dotnet, do not sleep for every put, what is the reason for this change?

Reply:
Looks better on the console, without the sleep it populates the map too fast and text becomes unreadable on console. It is a cosmetic change, duration of the sleep may be shortened, but I think it is better this way.